src/random_test.cpp

```diff
@@ -1638,11 +1685,32 @@ void Table::UpdateRandomROW(Thd1 *thd) {
       break;
     }
   }
+  if (prob <= 90)
+    sql = sql +
+         columns_->at(set)->rand_value() + " WHERE " +
```

Comment:
even you can move columns_->at(set)->rand_value() + " WHERE " in defination of sql.

Reply:
Done